extensions/amp-iframe/1.0/amp-iframe.js

```diff
@@ -27,7 +27,7 @@ class AmpIframe extends BaseElement {
    * element and hides it if it does. Also checks the position of the iframe on the
    * document.
    */
-  handleOnLoad_() {
+  handleOnLoad() {
```

Comment:
Does this need override?
```suggestion
  /** @override */
  handleOnLoad() {
```

Reply:
added